docs/pages/components/identifier.md

```diff
@@ -7,6 +7,8 @@ toc: false
 permalink: components/identifier.html
 folder: components
 ---
+> **DEPRECATED**. Use the [Avatar](avatar.html) component.
```

Comment:
[Avatar]({{site.baseurl}}/components/avatar.html)

Reply:
? this works